plutus-ledger-api/plutus-ledger-api.cabal

```diff
@@ -151,7 +151,6 @@ test-suite plutus-ledger-api-test
     Spec.CostModelParams
     Spec.Eval
     Spec.Interval
-    Spec.NoThunks
```

Comment:
Incorporated it into `Spec.Eval` where it shares code with a test I added in this PR.